### PR TITLE
bootstrap: hot fix bootstrap panic

### PR DESCRIPTION
**Pull request by tiancaiamao**

### CODE REVIEW

**Comment by zimulala** [comment]
LGTM

File: bootstrap.go [hunk 1]
```diff
@@ -481,7 +481,7 @@ func upgradeToVer13(s Session) {
 			log.Fatal(err)
 		}
 	}
-	mustExecute(s, "UPDATE mysql.user SET Create_tmp_table_priv='Y',Lock_tables_priv='Y',Create_view_priv='Y',Show_view_priv='Y',Create_routine_priv='Y',Alter_routine_priv='Y',Event_priv='Y'")
+	// mustExecute(s, "UPDATE mysql.user SET Create_tmp_table_priv='Y',Lock_tables_priv='Y',Create_view_priv='Y',Show_view_priv='Y',Create_routine_priv='Y',Alter_routine_priv='Y',Event_priv='Y'")
```
Comment (shenli) [inline bootstrap.go]: Why comment this? Is there something wrong with this update?

Comment (hanfei1991) [inline bootstrap.go]: This update raises a panic in gaea cluster.......

Comment (zimulala) [inline bootstrap.go]: PTAL @tiancaiamao

**Pull request closed without merging**